From e5b5e90331fc7ebaa840179237aa90c520a23e05 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey.doherty@gmail.com>
Date: Mon, 23 Jun 2014 13:10:19 +0100
Subject: [PATCH 2/5] session: Lie about desktop environment

Under certain configurations some tools may not work, such as xdg-su.
This commit lies about the desktop environment (setting it to GNOME) so
that tools like xdg-su under YaST/OpenSUSE work (fixes issue 10)
---
 session/budgie-session.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/session/budgie-session.c b/session/budgie-session.c
index e5ce503..9fd4988 100644
--- a/session/budgie-session.c
+++ b/session/budgie-session.c
@@ -139,6 +139,8 @@ gint main(gint argc, gchar **argv)
         GSimpleAction *action = NULL;
         int status = 0;
 
+        g_setenv("DESKTOP_SESSION", "gnome", TRUE);
+
         app = g_application_new(BUDGIE_SESSION_ID, G_APPLICATION_FLAGS_NONE);
         g_signal_connect(app, "activate", G_CALLBACK(activate), NULL);
 
-- 
1.9.1

